getting-started/GettingStarted.md

```diff
@@ -0,0 +1,48 @@
+# Getting Started: Publishing a Kubernetes App on Azure Marketplace
+
+### Introduction
+Thank you for joining us in our Kubernetes Apps public preview program. We’re delighted to have you participate in our vibrant and ever-growing ecosystem, where you can publish and monetize your Kubernetes application. This document has all the necessary steps to get you started and can enable you to publish your application in 1-2 weeks to the Kubernetes marketplace in Azure.
+
+### Pre-requisites
+You must meet the following criteria to be able to participate in the public preview:
```

Comment:
same here: Is term 'Public Preview' required? In your communication email, you can mention 'Public Preview'. 
This doc will be applicable post GA as well right?

Reply:
removed "public preview"